stashbot/bot.py

```diff
@@ -300,3 +294,12 @@ def _event_to_doc(self, conn, event):
             'server': conn.get_server_name(),
             'host': event.source.host,
         }
+
+    def _do_es_index(self, index, doc_type, body):
+        try:
+            return self.es.index(index=index, doc_type=doc_type, body=body,
+                                 consistency="one")
+        except elasticsearch.ConnectionError as e:
+            self.logger.exception(
+                'Failed to log to elasticsearch: %s', e.error)
+            return False
```

Comment:
This is going to cause a TypeError on line 254 for "argument of type 'bool' is not iterable" when it fails. I'll do something to change that in a followup commit.